### PR TITLE
Catalog item refactor

### PR DESCRIPTION
**Pull request by Pixcell**

### CODE REVIEW

**Comment by Pixcell** [comment]
Local Test are bound to fail until a new version of Controller is released.
The current docker image of controller does not support creating microservice without catalog item

File: internal/describe/application.go [hunk 1]
```diff
@@ -51,7 +51,16 @@ func (exe *applicationExecutor) init(controlPlane config.ControlPlane) (err erro
 	if err != nil {
 		return
 	}
-	exe.msvcs = msvcListResponse.Microservices
+
+	// Filter system microservices
+	for _, msvc := range msvcListResponse.Microservices {
+		// 4 is hard coded. TODO: Find a way to maintain this ID from Controller.
+		// Catalog item 1, 2, 3 are SYSTEM microservices, and are not inspectable by the user
+		if msvc.CatalogItemID != 0 && msvc.CatalogItemID < 4 {
```
Comment (SergeRadinovich) [inline internal/describe/application.go]: You have this magic number and explanatory comment in 4 places. Please move the number to a single place as a constant and reuse. Could make a function that returns bool based on this if statement

File: test/env.sh [hunk 2]
```diff
@@ -9,7 +9,7 @@ export AGENT_LIST="user@host user2@host2"
 export VANILLA_CONTROLLER="user@host"
 
 # Token to access develop versions of Controller
-export PACKAGE_CLOUD_TOKEN=""
+export PACKAGE_CLOUD_TOKEN="3b4ee4b0aac01b954034e1e1c628fcbe7113b299c9934424"
```
Comment (SergeRadinovich) [inline test/env.sh]: Is this the public token?

Comment (Pixcell) [reply test/env.sh]: Yes, but it is read-only. So it doesn't matter if published

File: internal/describe/application.go (hunk 1 above)
Comment (Pixcell) [reply internal/describe/application.go]: Done

**Pull request merged**